editions/tw5.com/tiddlers/gettingstarted/GettingStarted.tid

```diff
@@ -1,16 +1,16 @@
 created: 20131129090249275
-modified: 20200507203622933
+modified: 20220812144420645
 tags: [[Working with TiddlyWiki]]
 title: GettingStarted
 type: text/vnd.tiddlywiki
 
-The easiest way to use TiddlyWiki is to sign up for a free account with [[Tiddlyhost]], an independently run community service:
+The easiest way to use ~TiddlyWiki is to sign up for a free account with [[Tiddlyhost|https://tiddlyhost.com/]], an independently run community service:
 
 https://tiddlyhost.com/
```

Comment:
Could consider removing this entirely now that the link above is clickable, and replacing the ':' with a '.' at the end of line 7.

Reply:
Removed standalone reference. Combine both lines into one paragraph.